### ISSUE TITLE
Switch to paid Travis

### ISSUE THREAD

**Issue by wagerlabs** [aeternity/epoch#203]

**Comment by sennui** [comment]
Luka, I will sync with you when we get creds to the commercial travis account.

**Comment by uwiger** [comment]
Given that OSS projects can't switch to travis-ci.com, perhaps these links can be of some help?
https://make.wordpress.org/core/2017/05/18/increased-concurrent-jobs-on-travis-ci-builds/
https://blog.travis-ci.com/2014-07-18-per-repository-concurrency-setting/

**Comment by sennui** [comment]
This is response from travis re open source upgrade (it is possible!)

```

We've proceeded to refund your payments. You should see them appear on your credit card statement in the next 5 to 10 business days.

It is indeed possible to increase the number of jobs your organization can run for its open source projects. The normal, free package gives you five concurrent jobs (which equals five concurrent workers, so to speak). Beyond that, the next step up includes 5 extra concurrent jobs, effectively raising the limit to 10, for $250/month, paid annually and in advance.

Do you need more options, and is this something you'd generally like to proceed with?

Please keep us posted and we will follow-up upon your reply.
```

**Comment by lucafavatella** [comment]
> This is response from travis re open source upgrade (it is possible!)

@sennui A couple of notes to keep in mind:
* I assume at least some of those extra 5 workers are able to execute OSX jobs;
* Could latency of job start be reduced?

**Comment by wagerlabs** [comment]
We cannot switch a public project to paid Travis. The solution is to move from feature tickets in the company repo to feature tickets in repos forked to private accounts. Also, @sennui had a way to arrange to pay to upgrade our public Travis. There was something about prepaying for a year.

**Comment by adamnovak** [comment]
Hello,

Over at https://github.com/vgteam/vg/issues/1169 we have also tried (and seem to be failing) to pay Travis for more stuff for our public Github repo. In our case, we need longer build timeouts.

So far we've signed up for travis-ci.com for our organization, paid them, and had absolutely no benefit from it, because our repos don't show up over there, because they're public. I haven't been able to get a reply from Travis support on what to do about this for a couple of days.

@sennui, Who over at Travis is your contact for that $250/month more-concurrent-jobs plan for open source projects? I haven't found any public mention of it on their website.
